ako-operator/bundle/manifests/ako-operator.clusterserviceversion.yaml

```diff
@@ -479,7 +484,7 @@ spec:
               containers:
               - command:
                 - /ako-operator
-                image: projects.registry.vmware.com/ako/ako-operator@sha256:378c4bb64cfff509193930b7bdf981c299e306c4bebf6f5f3197255c5c0e38dd
+                image: projects.registry.vmware.com/ako/ako-operator@sha256:c556d055a9d6773c9a41655051f831dc83e548ee8d505a3794f1590fa6b8ce32
```

Comment:
`sha` will be same for 1.8.1 and 1.8.2? I think : divide it into 2 PR: One for AKO operator specific to 1.8.2 and then for bundle changes. So that we can update proper sha in bundle.

Reply:
I have not made any bundle changes for 1.8.2. The changes that are currently included are from cherry picked commits of 1.8.1. Once this PR is merged, I will raise another PR with bundle changes.